superset-frontend/src/views/components/LanguagePicker.tsx

```diff
@@ -76,7 +76,7 @@ export default function LanguagePicker(props: LanguagePickerProps) {
         >
           <StyledLabel className="f16">
             <i className={`flag ${languages[langKey].flag}`} />
-            <a href={languages[langKey].url}>{languages[langKey].name}</a>
+            <a href={languages[langKey].url}>{t(languages[langKey].name)}</a>
```

Comment:
Just to make sure, I hope having these translated doesn't cause an issue with people having a harder time switching to their preferred language!